### ISSUE TITLE
`analyzer` dependency is too old.

### ISSUE THREAD

**Issue by yohom** [vanlooverenkoen/kiwi#11]
Can you update the dependencies to newest? Thanks~

**Comment by zhangzhenxi** [comment]
```
Because kiwi_generator >=0.4.0 depends on analyzer >=0.32.4 <0.37.0 and sealed_unions_generator 0.0.2 depends on analyzer >=0.37.1 <0.38.2, kiwi_generator >=0.4.0 is incompatible with sealed_unions_generator 0.0.2.

And because no versions of sealed_unions_generator match >0.0.2 <0.1.0, kiwi_generator >=0.4.0 is incompatible with sealed_unions_generator ^0.0.2.

So, because flutter_app depends on both sealed_unions_generator ^0.0.2 and kiwi_generator ^0.4.0, version solving failed.
pub get failed (1; So, because flutter_app depends on both sealed_unions_generator ^0.0.2 and kiwi_generator ^0.4.0, version solving failed.)
```

**Comment by leen-nobbas** [comment]
@letsar 
Can you please update dependencies.

Because kiwi_generator >=0.4.0 depends on analyzer >=0.32.4 <0.37.0 and json_serializable 3.2.3 depends on analyzer >=0.37.1 <0.39.0, kiwi_generator >=0.4.0 is incompatible with json_serializable 3.2.3.
And because no versions of json_serializable match >3.2.3 <4.0.0, kiwi_generator >=0.4.0 is incompatible with json_serializable ^3.2.3.
So, because klient_flutter depends on both json_serializable ^3.2.3 and kiwi_generator ^0.4.0, version solving failed.
pub get failed (1; So, because klient_flutter depends on both json_serializable ^3.2.3 and kiwi_generator ^0.4.0, version solving failed.)

**Comment by vanlooverenkoen** [comment]
Fixed in kiwi 0.3.0 & kiwi_generator 0.5.0
